pkg/registry/core/event/strategy.go

```diff
@@ -58,7 +58,7 @@ func (eventStrategy) PrepareForUpdate(ctx context.Context, obj, old runtime.Obje
 
 func (eventStrategy) Validate(ctx context.Context, obj runtime.Object) field.ErrorList {
 	event := obj.(*api.Event)
-	return validation.ValidateEvent(event)
+	return validation.ValidateEventCreate(event, api.SchemeGroupVersion)
```

Comment:
shouldn't this be extracting the request groupVersion from the context?

Reply:
Done.